ansible/roles/operator-pipeline/tasks/community-pipeline-event-listener.yml

```diff
@@ -0,0 +1,63 @@
+---
+- name: Create event listener allowing triggering the community pipelines via Github webhook
+  tags:
+    - triggers
+  block:
+    - name: Create event listener
+      k8s:
+        state: present
+        namespace: "{{ oc_namespace }}"
+        definition:
+          apiVersion: triggers.tekton.dev/v1alpha1
+          kind: EventListener
+          metadata:
+            name: operator-pipeline-github-listener
```

Comment:
Let's use something like `community-operator-pipeline-github-listener` to differentiate from the ISV listener.